docs/guides/engineering/fresh-config.md

```diff
@@ -0,0 +1,78 @@
+# Ensuring Fresh Feature Flag Configurations
+
+When working with feature flags, there are certain use cases, like kill switches, where it can be important to ensure your application always has the most up-to-date configuration. This guide explains how to maintain fresh configurations across different platforms when using Eppo's SDKs.
+
+## Server-Side Applications
+
+Server-side applications automatically handle configuration freshness through built-in polling mechanisms. You can configure the polling interval during initialization of the SDK.
+
+## Client-Side Applications
+
+### JavaScript Client
+For browser-based applications, you have several options to ensure fresh configurations:
+
+1. **Enable Polling**
+```javascript
+const client = new EppoClient({
+    apiKey: 'your-api-key',
+    pollInterval: 30000 // Poll every 30 seconds
```

Comment:
This will set the polling interval but won't have it poll. replace with
`pollAfterSuccessfulInitialization: true`

(source: https://github.com/Eppo-exp/js-sdk-common/blob/main/src/client/eppo-client.ts#L87)